api/hrlh.rb

```diff
@@ -27,23 +36,41 @@ def database
         requires :email, type: String
       end
       post do
-        database['interviewer']
-        .set(
-            params[:email],
+        interview_db
+        .create(
             {
                 name: params[:name],
                 email: params[:email]
-            },
-            false
+            }
         ).value
       end
+
+      desc 'single interviewer'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.